js/playlists.track.js

```diff
@@ -302,7 +303,9 @@ $().ready(function(){
                 $(this).val(v);
 
             // if time is after midnight, set edate field to correct date
-            $("INPUT[name=edate]").val(val.toISOString().split('T')[0]);
+            // NOTE: commenting this out because it advances the edate when
```

Comment:
This needs to be there.  Is its removal left over from #158, which turned out to be something else?

If you are seeing edate change unexpectedly, please give a specific case to reproduce.

Reply:
will restore it but i suggest that the adjustment be moved into the server code. it took quite a while to find where this was being done.